core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -769,7 +772,14 @@ private[spark] object AppStatusStore {
   def createLiveStore(
       conf: SparkConf,
       appStatusSource: Option[AppStatusSource] = None): AppStatusStore = {
-    val store = new ElementTrackingStore(new InMemoryStore(), conf)
+    val storePath = conf.get(LIVE_UI_LOCAL_STORE_DIR).map(new File(_))
+    // For the disk-based KV store of live UI, let's simply make it ROCKSDB only for now,
+    // instead of supporting both LevelDB and RocksDB. RocksDB is built based on LevelDB with
+    // improvements on writes and reads. Furthermore, we can reuse the RocksDBFileManager in
+    // streaming for replicating the local RocksDB file to DFS. The replication in DFS can be
+    // used for Spark history server.
```

Comment:
nit: Can we remove the part about replication for now ? When we add support for it, we can suitably modify the comment.

Reply:
Thanks, I have removed the comment about replication.